CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.2.0
+  - Support adding tag(s) on DNS lookup times out, defaults to `["_dnstimeout"]` [#67](https://github.com/logstash-plugins/logstash-filter-dns/pull/67)
```

Comment:
change to logging level not referenced in the changelog

Reply:
Thanks, updated.